src/libmongoc/src/mongoc/mongoc-stream-tls-openssl.c

```diff
@@ -853,6 +815,81 @@ mongoc_stream_tls_openssl_new (mongoc_stream_t *base_stream, const char *host, m
    RETURN ((mongoc_stream_t *) tls);
 }
 
+/*
+ *--------------------------------------------------------------------------
+ *
+ * mongoc_stream_tls_openssl_new --
+ *
+ *       Creates a new mongoc_stream_tls_openssl_t to communicate with a remote
+ *       server using a TLS stream.
+ *
+ *       @base_stream should be a stream that will become owned by the
+ *       resulting tls stream. It will be used for raw I/O.
+ *
+ *       @trust_store_dir should be a path to the SSL cert db to use for
+ *       verifying trust of the remote server.
```

Comment:
```suggestion
```

Though not introduced in this PR, suggest removing non-applicable comment. There is no `trust_store_dir` argument.